packages/material-ui-lab/package.json

```diff
@@ -52,7 +52,7 @@
     "@material-ui/utils": "^5.0.0-alpha.14",
     "clsx": "^1.0.4",
     "prop-types": "^15.7.2",
-    "react-is": "^16.8.0"
+    "react-is": "^16.8.0 || ^17.0.0"
```

Comment:
This feels a bit awkward as a direct dependency but should "just work" in practice. More details why this is technically wrong can be found in https://github.com/facebook/react/issues/20099.